tools/dockerfile/interoptest/grpc_interop_php7/Dockerfile

```diff
@@ -75,6 +75,9 @@ RUN mkdir /var/local/jenkins
 RUN curl -sS https://getcomposer.org/installer | php
 RUN mv composer.phar /usr/local/bin/composer
 
+# Install valgrind for memory tests
+RUN apt-get update && apt-get install -y     valgrind
```

Comment:
you really shouldn't need  valgrind for the interop tests. Actually,  where's the addition for the basic (non-interop) test image for PHP?